pyopenms_viz/testing/BokehSnapshotExtension.py

```
@@ -107,19 +107,29 @@ def compare_json(json1, json2):
-                            return True
-                    print(f"Element {i} not in second list")
-                    return False
+                    if (
+                        "type" not in i.keys()
+                    ):  # if "type" not present than dictionary with only id, do not need to compare, will get key error if check
+                        check = False
+                        pass
+                    if check:  # find corresponding entry in json2 only if check is true
+                        for j in json2:
+                            if (
+                                "type" not in j.keys()
```

Ruff (0.8.2)

Use `key not in dict` instead of `key not in dict.keys()`

Remove `.keys()`

(SIM118)